src/libs/actions/IOU.ts

```diff
@@ -712,6 +712,7 @@ function buildOnyxDataForMoneyRequest(
             value: {
                 pendingAction: null,
                 pendingFields: clearedPendingFields,
+                routes: null,
```

Comment:
NAB: Maybe we should add a comment in at least one of these places to explain why we're doing this? We're clearing the route so that we use the distance saved in the proper unit, which is stored under the transaction custom unit once the request is created. The route is also not saved in the backend so we can't rely on it sticking around.

Maybe you can phrase it better. The point is that the reasoning behind this is pretty confusing and involved, so we should try to capture some of that.

Reply:
Added comment